src/Quadrature/quadrature.jl

```diff
@@ -109,7 +127,7 @@ for dim in 2:3
             for p in 1:size(data, 1)
                 points[p] = Vec{$dim,T}(@ntuple $dim i -> data[p, i])
             end
-            weights = data[:, $dim + 1]
+            weights = T.(data[:, $dim + 1])
```

Comment:
I suppose this is to support Float32. 
This doesn't seem to be tested?

Reply:
Good catch. I thought this paths are already tested, because we had the option to set the type already before this PR. Will add it.